css/styles.css

```diff
@@ -0,0 +1,94 @@
+* {
```

Comment:
Kindly try to match the Figma design for the mobile version you can achieve this by:
- Please remove the width for the `h3` that takes class `desc`.
- Kindly remove the height for the `h1` that takes class `hey-there`.
- Please add the `background-image` that exists in Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/158651824-593f9b1b-9f9e-40a4-8d07-9dd83b2b056e.png) | ![Template 3-Main-Mobile - Copy](https://user-images.githubusercontent.com/24703171/158652387-1e4c9f5a-0b9e-493c-8407-49d293461f9a.png) |